Makefile-os

```diff
@@ -8,12 +8,11 @@ export DOCKER_BUILDER=container
 DOCKER_TAG := addons-server-test
 DOCKER_PLATFORM := linux/amd64
 DOCKER_PROGRESS := auto
-DOCKER_COMMIT := $(shell git rev-parse HEAD)
+export DOCKER_COMMIT := $(shell git rev-parse HEAD || echo "commit")
 DOCKER_CACHE_DIR := docker-cache
 
 export DOCKER_VERSION ?= local
-export DOCKER_COMMIT ?= $(shell git rev-parse --short HEAD)
-export VERSION_BUILD_URL ?= local
+export VERSION_BUILD_URL ?= build
```

Comment:
mmm, what does this change do?

Reply:
Makes it clear that the default value for version is "local" and "build is "build"

When you don't have any specified arguments

old:

```json
{
"version": "local",
"build": "local"
}
```

new:

```json
{
"version": "local",
"build": "build"
}
```

It just removes that tiny little edge case where you don't know if both values are set to the same value or you are only using one variable.

For the removal of DOCKER_COMMIT, it's actually a dupe. the end result would be if git isn't available (sometimes the case in the container) your commit would be "commit"